src/lib/ctable.lua

```diff
@@ -110,15 +120,22 @@ local optional_params = {
    hash_fn = false,
    initial_size = 8,
    max_occupancy_rate = 0.9,
-   min_occupancy_rate = 0.0
+   min_occupancy_rate = 0.0,
+   salt = get_random_salt()
```

Comment:
This default is set globally -- all hash tables in this process will default to this salt value.  Probably not what you want.